apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.ts

```diff
@@ -643,4 +702,116 @@ export class AdvanceReceiptReportComponent implements AfterViewInit, OnDestroy,
             this.advanceSearchModel.unusedAmountFilter.filterValues = this.advanceReceiptAdvanceSearchAmountFilters;
         }
     }
+
+    /**
+     * This will update the search query param
+     *
+     * @param {*} value
+     * @memberof AdvanceReceiptReportComponent
+     */
+    public updateSearchQuery(value): void {
+        this.searchQueryParams.q = value;
+    }
+
+    /**
+     * This will preview voucher
+     *
+     * @param {*} receipt
+     * @memberof AdvanceReceiptReportComponent
+     */
+    public previewVoucher(receipt: any): void {
+        // if (this.voucherApiVersion === 2) {
```

Comment:
⚠️ Please check method has commented code in it

Reply:
yes we are not allowing preview/edit currently.